### ISSUE TITLE
Re-Permissioning Campaign - Email Blast #1 (Feb 2024)

### ISSUE THREAD

**Issue by dsgit215** [chrisflesher/rcvfortexas#111]
Campaign to encourage existing supporters to complete their communication preferences.

**Comment by dsgit215** [comment]
Debbie and Paula Winter to meet 2024.01.26 to discuss.

**Comment by dsgit215** [comment]
- Harriet Wasserstrum and David Green approved email content developed by Debbie and Paula  - via email communications week of January 29.
- Debbie created filter to use in email blast - Re-Permissioning Campaign 2024.   See image for filter details. 

![image](https://github.com/rcvfortexas/rcvfortexas/assets/77990608/0dabef46-1d6b-4fa4-b05a-ccdd709bbe72)

- Scheduled email blast named "Re-Permissioning Feb 2024" to be sent Tuesday, Feb 6, 2024 @ 2:10 PM central.

I'll keep this issue is open until after the email is sent and initial results can be observed.

**Comment by dsgit215** [comment]
Status:

Overall and to date, there are 3,153 records in that database that have an email address that is not "bad", not bouncing, and where the individual has opted in to general email communications.  Out of this, 91 have provided their communication preferences (i.e. "consent" responses).  

Here are some of the stats associated with the re-permissioning email sent in February:
Email sent to 3,158 individuals
Open rate: 35.87% (This loosely aligns with Newsletter open rates.)
Clicked a link: 2.56%
Unsubscribed: 0.82%
Unfortunately, there's not a way (that I have been able to find in NationBuilder) to attribute the completion of the consent form directly to an email.

Nevertheless, I recommend that we continue to periodically remind subscribers to provide their communication preferences via email blasts and the newsletter.

Closing this issue and opening a new issue for the 2nd email blast (https://github.com/rcvfortexas/rcvfortexas/issues/116).
